docs/oid-info.md

```diff
@@ -189,27 +189,27 @@ that Sigstore operates.
+| job_workflow_sha   | ci_config_sha ([WIP][gitlab-wip-cliams]) | ??       | ?? | N/A        | Build Signer Digest          | An immutable reference to the specific version of the build instructions that is responsible for signing. Should include the digest type followed by the digest, e.g. `sha1:abc123`.   |
+| runner_environment | runner_environment | ??       | ??        | runner_environment        | Runner Environment           | For platforms to specify whether the build took place in platform-hosted cloud infrastructure or customer-hosted infrastructure. For example: `platform-hosted` and `self-hosted`.     |
+| server_url + repository         | server_url + project_path | ??       | ??        | scm_repo_url        | Source Repository URI                   | Should include a fully qualified repository URL.                                                                                                                                       |
+| sha                | sha | ??       | build_commit | N/A | Source Repository Digest                | An immutable reference to a specific version of the source code. Should include the digest type followed by the digest, e.g. `sha1:abc123`.                                            |
+| ref                | ref     | ??       | build_branch | scm_ref | Source Repository Ref                   | The source ref that the build run was based upon. For example: refs/head/main.                                                                                                         |
+| repository_id         | project_id     | ??       | ?? | N/A        | Source Repository Identifier     | Stable identifier for the owner of the source repository.                                                                                                                                      |
+| server_url + repository_owner         | server_url + namespace_path | ??       | ?? | N/A        | Source Repository Owner URI     | Fully qualified URL for the owner of the source repository.                                                                                                                                     |
+| repository_owner_id         | namespace_id     | ??       | ?? | N/A        | Source Repository Owner Identifier     | Stable identifier for the owner of the source repository.                                                                                                                                   |
+| server_url + workflow_ref       |  ci_config_ref_uri ([WIP][gitlab-wip-cliams]) | ??       | ??        | platform_url + /build/ +  workflow_id        | Build Config URI             | A reference to the initiating build instructions.                                                                                                                                      |
```

Comment:
I wonder if this should be pipeline_id? 🤔 

My current understanding is the pipeline_id stores the configuration, and workflow_id is a run of the pipeline.

If not, would another field be more appropriate?

Reply:
Changed it to the API endpoint in Codefresh that returns the pipeline config, as the UI endpoint is a bit awkward as it requires a lot of query parameters. But the API endpoint should be completely valid and it uses the pipeine_id claim.